modules/services/easyeffects.nix

```
@@ -0,0 +1,60 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.easyeffects;
+
+  presetOpts = optionalString (cfg.preset != "") "--load-preset ${cfg.preset}";
+
+in {
+  meta.maintainers = [ maintainers.fufexan ];
+
+  options.services.easyeffects = {
+    enable = mkEnableOption ''
+      Easyeffects daemon.
+      Add `services.dbus.packages = with pkgs; [ gnome.dconf ];`
+      to /etc/nixos/configuration.nix for daemon to work correctly
```

How does this get rendered? I think it needs to be something like:

```suggestion
      Easyeffects daemon.
      Add <literal>services.dbus.packages = with pkgs; [ gnome.dconf ];</literal>
      to <filename>/etc/nixos/configuration.nix</filename> for daemon to work correctly
```

---

Right, an oversight on my end again.

The definition should use `<code>` instead of `<literal>`.
`<option>` cannot be used because this is an external option.

---

I think `programlisting` is nice for longer lines of code. Perhaps something like
```xml
Note, it is necessary to add
<programlisting language="nix">
services.dbus.packages = with pkgs; [ gnome.dconf ];
};
</programlisting>
to your system configuration for the daemon to work correctly.
```

---

I wasn't sure whether to include that `};`, as I thought it was a typo. Feel free to correct me if that's not the case.